lib/pyspark_init.py

```diff
@@ -11,30 +20,108 @@ def create_spark_context() -> SparkSession:
     spark = SparkSession.builder.appName("Colab").getOrCreate()
     return spark
 
+map_types = {
+    str : StringType(),
+    int : IntegerType(),
+    bool : BooleanType(),
+    float: FloatType(),
+    'timestamp' : TimestampType(),
+    'date' : DateType(),
+    List[str] : ArrayType(StringType()),
+    Tuple[str] : ArrayType(StringType()),
+    Dict[str, str] : MapType(StringType(), StringType())
+}
+
+column_conf = {
+  'title_basics' : ['tconst','titleType','primaryTitle','originalTitle','isAdult','startYear','endYear','runtimeMinutes','genres'],
+  'principals' : ['tconst','ordering','nconst','category','job','characters'],
+  'name_basics' : ['nconst','primaryName','birthYear','deathYear','primaryProfession','knownForTitles'],
+  'oscars': ['year_film', 'year_ceremony', 'ceremony', 'category', 'name', 'film', 'winner'],
+  'globe':['year_film', 'year_award', 'ceremony', 'category', 'nominee', 'film', 'win'],
+  'emmy_awards': ['id', 'year', 'category', 'nominee', 'staff', 'company', 'producer', 'win']
+}
+column_type_collection = {
+    int : ['startYear', 'endYear', 'runtimeMinutes', 'birthYear', 'deathYear', \
+          'isAdult', 'year_award', 'year_film', 'year_ceremony', 'ceremony', 'year'],
+    str : [ 'titleType', 'primaryTitle', 'originalTitle', 'genres', 'category', 'job', 'characters', \
+           'primaryName', 'primaryProfession', 'knownForTitles', 'nominee', 'film', 'category', 'name', 'staff','company', 'producer'],
+    bool : ['win', 'winner'],
```

Comment:
Z tego co pamiętam to kolumna `'isAdult'` przyjmuje tylko wartości 0 i 1. Zmieniłbym jej typ na `bool`

Reply:
(komentarz niżej)